Writerside/topics/EventSource.md

```diff
@@ -0,0 +1,84 @@
+# EventSource
+
+An event source provides a translation layer between your <tooltip term="game server">game server</tooltip>, the <tooltip term="backend platform">backend platform.</tooltip> and <tooltip term="fleet manager">fleet manager</tooltip>.
+This way your game only needs to interface with the sidecar, and it can be deployed anywhere.
+
+Currently, Catena supports the following event sources:
+
+<snippet id="event-source-list">
+<ul>
+<li>Catena Backend Event Source - An event source which matches the <a href="Server-Manager.md">Catena Server Manager</a> API.</li>
+<li>Virtual Server Event Source - A mock server for simulating calls or doing zero-code implementations</li>
+</ul>
+</snippet>
+
+## Configuration
+
+Below is an example configuration for an event source in the sidecar.
+
+```json lines
+{
+  "app": {
+      "event_source": {
+        "use": "catena_api_event_source",
+        "configs": {
+          "catena_api_event_source": {
+            "GrpcListenAddr": ":8080",
+            "HttpListenAddr": ":8085"
+          },
+          "virtual_server_event_source": {
+            "strategy": "continuous_strategy",
+            "continuous_strategy": {
+              "RequestInterval": "5s"
+            }
+          }
+        }
+      },
+    }
+}
+```
+
+`use` - The event source to use to ingest events into the sidecar, must match one of the configurations specified in `configs`.
+
+`configs` - The configurations for the event sources.
+
+[//]: # () TODO This will eventually be modified to support multiple event sources. These docs need to be updated when that changes
+
+## Catena API Event Source
```

Comment:
This is previously referred to as the "Catena Backend Event Source" but is named the "Catena API Event Source" here.